index.html

```diff
@@ -0,0 +1,198 @@
+        <h3>"Hello!Sharing World"</h3>
+        <h1>Creative commons <br> Summit<br>2015</h1>
+        <p>A joyful celebration believing in the value of openness and sharing, creating a postive  change with people from over 80 countries is taking place in october, in Korea.
+        </p>
+        <time datetime="2022-01-11T13:53">2015.10.15(THU) ~ 16(FRI)</time>
+        <h5>@ National Museum of Korea, Art Centre Nabi and more</h5>
+      </div>
+    </section>
+    <section class="programs">
+      <div class="center-container">
+        <h2>Main Program</h2>
+        <hr class="programLine" />
+      </div>
+      <div class="progContainer">
+        <div class="programCards">
+          <i class="fa fa-users fa-2x"></i>
+          <div><h3>Lecture</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-rocket fa-2x"></i>
+          <div><h3>Exhibition</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-comments-o fa-2x"></i>
+          <div><h3>Forum</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-microphone fa-2x"></i>
+          <div><h3>Workshop</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-graduation-cap fa-2x"></i>
+          <div><h3>Ignite</h3></div>
+          <div class="cardsColumn">
+            <p>
+              Many national governments around the world are showing themselves
+              to be inefficient
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="allPrograms">
+        <h3>JOIN THE EVENT</h3>
+        <hr />
+      </div>
+    </section>
+    <section class="speakers">
+      <div class="center-container">
+        <h2>Featured Speakers</h2>
+        <hr class="programLine" />
+      </div>
+      <div class="speakersContainer" id="container">
+        
+      </div>
+    </section>
+    <section class="partners">
+      <div class="partnersContainer">
+        <div class="mainTitle center-container">
+          <h2>Partners</h2>
+          <hr class="programLine" />
+        </div>
+
+        <div class="partnersMain">
+          <div class="partnersItem">
+            <img
+              src="../capston-project/imgs/intel.png"
+              width="250"
+              height="70"
+              alt="Open AI"
+              class="partner_img"
+            />
```

Comment:
- [ ] One of your partners is missing 🕵🏻‍♂️, I mean one image is not working but with that, We tested that alt is working well. Also, fix this issue and your project will be completed 🍀

<img width="816" alt="Screen Shot 2021-12-25 at 5 22 50 PM" src="https://user-images.githubusercontent.com/69011963/147384771-231c9125-fede-4bd6-bdc3-afe52f0b3083.png">

Reply:
thank you